magento/magento2
# Fix wrong position of button list when load "New Attribute" page

### Description (*)


Let check this gif:

![wrong_position](https://user-images.githubusercontent.com/8234209/78451079-7d30ee80-76ad-11ea-8ce5-3efc7a54d643.gif)

We can see, after 1 seconds, the button list re-arranged. 

![image](https://user-images.githubusercontent.com/8234209/78451143-df89ef00-76ad-11ea-8b27-69cc58112f52.png)

to

![image](https://user-images.githubusercontent.com/8234209/78451145-e57fd000-76ad-11ea-8e79-c5af70382ef8.png)


### Related Pull Requests


### Fixed Issues (if relevant)


### Manual testing scenarios (*)

1. Go to backend, create new product
2. Fill required field, go to "Create Configurations"
3. Create New Attribute
4. Look at the position of the button "Reset" and "Save Attribute"
5, It should be:

![image](https://user-images.githubusercontent.com/8234209/78451150-eadd1a80-76ad-11ea-8e4a-8d16f1ed8139.png)


### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29597: Fix wrong position of button list when load "New Attribute" page
